### DISCUSSION TITLE
No method in multimethod 'connection-details->spec' for dispatch value: :db2

### DISCUSSION THREAD

**Discussion by antoniocarlosbotelho** [alisonrafael/metabase-db2-driver#6] [General]
![image](https://user-images.githubusercontent.com/74840997/230104061-a5844f9c-9a23-4517-b410-21933d2cce78.png)

after updating metabase  v0.46.0, it shows this error, could you help the community to include it with the new version of metabase? would solve the variable date filter problem.

**Comment by alisonrafael** [comment]
Hi!

The last version of this driver [v1.1.44](https://github.com/alisonrafael/metabase-db2-driver/tree/v1.1.44) ([jar](https://github.com/alisonrafael/metabase-db2-driver/releases/download/v1.1.44/db2.metabase-driver.jar)) works good with Metabase [0.45.2.1](https://github.com/metabase/metabase/releases/tag/v0.45.2.1) ([jar](https://downloads.metabase.com/v0.45.2.1/metabase.jar)), only do not work filter dates by week and quarter.

I'm trying to update the driver to Metabase v0.46.0 but with no success yet.

**Reply by antoniocarlosbotelho** [reply]
![image](https://user-images.githubusercontent.com/74840997/230188500-97466894-796b-40fa-b8a1-a2f23dbcf569.png)
In version 0.45.2.1 the monthly dates cannot be found in your drive, is there a missing parameter?

**Reply by alisonrafael** [reply]
you are right! i will try fix this in the next version

**Reply by antoniocarlosbotelho** [reply]
thank you for your excellent work

**Reply by antoniocarlosbotelho** [reply]
with the new version of your plugin everything works fine with the date filters too thanks
